pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -405,75 +403,6 @@ def test_get_loc_key_unit_mismatch_not_castable(self):
 
         assert key not in dti
 
-    @pytest.mark.parametrize("method", [None, "pad", "backfill", "nearest"])
-    @pytest.mark.filterwarnings("ignore:Passing method:FutureWarning")
-    def test_get_loc_method_exact_match(self, method):
-        idx = date_range("2000-01-01", periods=3)
-        assert idx.get_loc(idx[1], method) == 1
-        assert idx.get_loc(idx[1].to_pydatetime(), method) == 1
-        assert idx.get_loc(str(idx[1]), method) == 1
-
-        if method is not None:
-            assert idx.get_loc(idx[1], method, tolerance=pd.Timedelta("0 days")) == 1
-
-    @pytest.mark.filterwarnings("ignore:Passing method:FutureWarning")
-    def test_get_loc(self):
-        idx = date_range("2000-01-01", periods=3)
-
-        assert idx.get_loc("2000-01-01", method="nearest") == 0
```

Comment:
ive got a branch that started in on this, but i found that when I changed these to the suggested `idx.get_indexer(["2000-01-01"], method="nearest")[0]` a whole bunch of problems cropped up

Reply:
Ah okay. I'm happy to close this PR if you've dug into this further

Reply:
I'll close for now since you've dug into this a little more